registrations/tasks.py

```diff
@@ -392,6 +391,48 @@ def run(self, registration_id, **kwargs):
 validate_subscribe = ValidateSubscribe()
 
 
+@app.task()
+def remove_personally_identifiable_fields(registration_id):
+    """
+    Saves the personally identifiable fields to the identity, and then
+    removes them from the registration object.
+    """
+    registration = Registration.objects.get(id=registration_id)
+
+    fields = set((
+        'id_type', 'mom_dob', 'passport_no', 'passport_origin', 'sa_id_no',
+        'language', 'consent')).intersection(registration.data.keys())
```

Comment:
Is 'language' not something we'd want to keep for aggregated or demographic data? Especially for investigations like the ones we're currently doing